ui/StatusQ/src/StatusQ/Popups/StatusSearchPopupMenuItem.qml

```diff
@@ -14,6 +14,7 @@ MenuItem {
     property StatusAssetSettings assetSettings: StatusAssetSettings {
         height: 16
         width: 16
+        isImage: (name.toString() !== "")
```

Comment:
Seeing how many times this pattern is repeated all over the codebase... this should be part of the StatusAssetSettings, wdyt?

Reply:
@caybro I know, we have an issue for this https://github.com/status-im/status-desktop/issues/7550